chore: docs grammar fixes (#8)
vercel/workflow: docs/content/docs/foundations/workflows-and-steps.mdx

- Automatic retry on errors
- Results persisted for replay

By default, steps have a maximum of 3 retry attempts before they fail and propagate the error over to the workflow. Learn more about errors and retrying in the [Errors & Retrying](/docs/foundations/errors-and-retries) page.

<Callout type="info">
Step functions are primarily meant to be used inside a workflow.
</Callout>

Calling a step from outside a workflow, or from another step will essentially run the step in the same process like a normal function (in other words, the `use step` directive is a no-op).

---

### Suspension and Resumption

Workflow functions have the ability to automatically suspend while they wait on asynchronous work. While suspended, the workflow's state is stored via the event log and no compute resources are used until the workflow resumes execution.

> There is more than one way for a workflow to suspend:

Reviewer: good eye, might i suggest this?

```suggestion
There are multiple ways a workflow can suspend:
```
